native_client/BUILD

```diff
@@ -18,6 +18,19 @@ KENLM_INCLUDES = [
     "kenlm",
 ]
 
+DECODER_SOURCESWIN = glob([
```

Comment:
I guess we should avoid that, and factorize between windows / unix sources, the only difference is OpenFST in fact.